cli/pkg/cmd/create/create.go

```diff
@@ -43,7 +42,14 @@ func create(cmd *cobra.Command, args []string) {
 		cfgDir = args[0]
 	}
 
-	dir, err := workspace.Create(cfgDir)
+	w := console.GetMiddleware(func(input string) string {
+		return color.Colorize(color.Cyan, "#\t"+input)
+	})
+	provider := providers.MustGet(providers.ProviderName(providerName), w)
+	overrides := provider.GetOverrides()
```

Comment:
Instead of passing the overrides as `--set` flags we merge them into the `airy.yaml` first before using it as a values file in Helm.